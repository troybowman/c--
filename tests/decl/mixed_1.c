void func1(int x, char y)
{
  char array[];
  char array[45];
}

char array[35];

void func2(void) {}
void func3(void) {}
void func4(void) {}
void func5(void) {}
void func6(void) {}

void func7(void)
{
  char func8(void);
  char func8;
  char func9[];
  char func9;
}

/* recoverable redeclaration errors */
int func2(char x), func3(char x), func4(char x), func5(char x), func6(int buf[]);

int func7;

()
  just
some junk


void missing_open_bracket(int x)

  asdflkhjasd
  asdflkjhasd
}

void func8()
{
  intad
    kjasdhkljh
    klajsd
    hasdfalk
  43453
}

/*
func8(void)
{
  this too
}

void func8(void)
{
  int x;
}

int array(void);

int array(void)
{

}
*/

this doesn't matter
we terminate after a function
definition is invalidated
