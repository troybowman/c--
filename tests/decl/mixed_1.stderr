syntax error, line: 3
syntax error, line: 17
syntax error, line: 19
error: function func2 redeclared at line 24 (previous declaration at line 9)
error: function func3 redeclared at line 24 (previous declaration at line 10)
error: function func4 redeclared at line 24 (previous declaration at line 11)
error: function func5 redeclared at line 24 (previous declaration at line 12)
error: function func6 redeclared at line 24 (previous declaration at line 13)
error: variable func7 redeclared at line 26 (previous declaration at line 15)
syntax error, line: 28
syntax error, line: 39
syntax error, line: 41
syntax error, line: 67
c-- returned nonzero exit code